### PR TITLE
Created cooperation notes container

### PR DESCRIPTION
**Pull request by dmtrth25**
Implemented notes container with animation and Drawer

https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/fd28b267-44aa-4554-afcd-d757a63ed435

**Resolved: #1560**

### CODE REVIEW

**Review by OlyaKorchan — APPROVED** [review]

File: src/containers/my-cooperations/cooperation-details/CooperationDetails.styles.ts [hunk 1]
```diff
@@ -1,20 +1,40 @@
...
-    color: palette.basic.blueGray,
-    mb: '20px'
-  },
-  notes: {
-    py: '10px',
-    color: 'inherit'
+    mb: '10px',
+    color: isNotesOpen ? palette.basic.lightBlue : palette.basic.blueGray,
+    transform: isNotesOpen ? 'translateX(-200px)' : 'translateX(0)',
+    transition: 'transform 0.3s ease-in-out, color 0.3s ease-in-out',
```
Comment (Tolik170) [inline src/containers/my-cooperations/cooperation-details/CooperationDetails.styles.ts:19]: Try to use out <AppDrawer /> component for notes container, and you will avoid transition and some other styles

File: src/containers/my-cooperations/cooperation-notes/CooperationNotes.styles.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import palette from '~/styles/app-theme/app.pallete'
+
+export const styles = {
+  divider: {
+    mx: '16px',
+    border: `1px solid ${palette.primary[200]}`
+  },
+  notesWrapper: (isNotes: boolean) => ({
+    width: isNotes ? { md: '400px', sm: '380px' } : '0px'
```
Comment (Tolik170) [inline src/containers/my-cooperations/cooperation-notes/CooperationNotes.styles.ts:9]: same here, with drawer I guess you will not need isNotes param

File: src/styles/app-theme/custom-mui.styles.ts [hunk 3]
```diff
@@ -27,3 +27,15 @@ export const theme = createTheme({
     MuiTableRow: table
   }
 })
+
+export const breakpointsTheme = createTheme({
```
Comment (Tolik170) [inline src/styles/app-theme/custom-mui.styles.ts:31]: Do you need to override mui default breakpoints, I think it doesn`t have a lot of sense

**Review by Mav-Ivan — APPROVED** [review]

File: src/containers/my-cooperations/cooperation-details/CooperationDetails.styles.ts (hunk 1 above)
Comment (dmtrth25) [reply src/containers/my-cooperations/cooperation-details/CooperationDetails.styles.ts]: > Try to use our 'AppDrawer' component for notes container, and you will avoid transition and some other styles

used AppDrawer

File: src/containers/my-cooperations/cooperation-notes/CooperationNotes.styles.ts (hunk 2 above)
Comment (dmtrth25) [reply src/containers/my-cooperations/cooperation-notes/CooperationNotes.styles.ts]: > same here, with drawer I guess you will not need isNotes param

used AppDrawer instead isNotes param

File: src/styles/app-theme/custom-mui.styles.ts (hunk 3 above)
Comment (dmtrth25) [reply src/styles/app-theme/custom-mui.styles.ts]: > Do you need to override mui default breakpoints, I think it doesn`t have a lot of sense

removed breakpointsTheme. 
I already use AppDrawer for less screen sizes

**Review by Tolik170 — APPROVED** [review]

**Pull request merged**